README.md

```diff
@@ -206,6 +206,44 @@ e.g. kill -QUIT PROCESS_ID
 ```
 
 
+Docker Container
+----------------
+**Build the container image**  
+Install [Go](https://golang.org/dl/) and [Docker](https://www.docker.com/get-docker), then just:
+
+    $ make
+
+This will create a `linio/rabbitmq-worker` image.
+
+**Run the container**  
+To run the container and redirect the log files to stdout and stderr:
+1) Set the stdout/stderr file paths in the configuration file:
+
+```
+[Log]
+LogFile = /dev/stdout
+ErrFile = /dev/stderr
+```
+
+2) Attach stdout/stderr from the container, and mount the configuration file as a data volume:
+
+
+    $docker run -a stdout -a stderr -v $(pwd)/rabbitmq-worker.conf:/etc/rabbitmq-worker.conf linio/rabbitmq-worker > worker.log 2> worker.err &
```

Comment:
What's the reason for the complexity of the command? Why not just `$ docker run -v $(pwd)/rabbitmq-worker.conf:/etc/rabbitmq-worker.conf linio/rabbitmq-worker`?

Also, make sure to add a space after the `$` in all the commands to symbolize it's a command and not a variable.